### PR TITLE
Add Header to the page

### PR DESCRIPTION
**Pull request by MohamedCK**
- Created navbar for the desktop
- Added image loader for webpack
- styled navbar

### CODE REVIEW

**Comment by MohamedCK** [comment]
#47

File: src/styles/style.css [hunk 1]
```diff
@@ -1,3 +1,59 @@
-.greeting {
-  color: blue;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```
Comment (tinatsina) [inline src/styles/style.css:5]: ## Teamwork issues
- adding defaults across all files, disturbs other developers' file systems.
- contain your styling references to just the header section, please.
- as other developers will work on the `footer` and `main-page`

File: src/index.html [hunk 2]
```diff
@@ -1,14 +1,40 @@
...
+    </div>
+
+    <div class="desktop-nav">
+      <a href="">Home</a>
+      <a href="">Recipes</a>
+      <a href="">Ingredient</a>
+    </div>
+
+  </header>
+  <script type="module" src="index.js"></script>
```
Comment (tinatsina) [inline src/index.html:37]: ## Script import issues
- The script can be automaticaly loaded by Webpack.
- No need to add it yourself
- you can remove this section of the code

**Review by tinatsina — CHANGES_REQUESTED** [review]
## Hi CAKE 👋 

## Status: Changes required 🟡 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

K & P So Close

## Highlights
- No Linter errors ✅
- Well file structure ✅
- Good summary description ✅

## Required Changes ♻️
- [x] I suggest not using `global` CSS defaults as other developers are working on the front end too.

- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: src/index.html (hunk 2 above)
Comment (tinatsina) [reply src/index.html]: ## Resolved 👍 

- removed the imported script and left `Webpack` to handle the imports
- Errors gone now

**Pull request merged**